src/ls.js

```diff
@@ -9,6 +9,7 @@ var _pwd = require('./pwd');
 //@ ### ls([options,] path_array)
 //@ Available options:
 //@
+//@ + `-l`: long list format
```

Comment:
I'd recommend being a little more verbose here especially if you stick with using Object.create below...